x/data/server/testsuite/suite.go

```diff
@@ -35,9 +35,7 @@ func (s *IntegrationTestSuite) SetupSuite() {
 	s.ctx = s.fixture.Context()
 	s.msgClient = data.NewMsgClient(s.fixture.TxConn())
 	s.queryClient = data.NewQueryClient(s.fixture.QueryConn())
-	if len(s.fixture.Signers()) < 2 {
-		s.FailNow("expected at least 2 signers, got %d", s.fixture.Signers())
-	}
+	s.Require().GreaterOrEqual(len(s.fixture.Signers()), 2)
```

Comment:
I know it's not something new as of this PR... but can you explain briefly why at least 2 signers are necessary in the x/data test suite?

Reply:
It's just the number of signers we use in test cases. If the test cases were written differently this would be different.